release-notes/6.0/preview/api-diff/preview7/Asp.Net/6.0-preview7_System.Security.Cryptography.md

```diff
@@ -0,0 +1,617 @@
+# System.Security.Cryptography
+
+``` diff
+-namespace System.Security.Cryptography {
```

Comment:
What was all this?

Reply:
I think it's that the CNG assembly was part of the asp shared runtime.  Now that we've just put it in the core shared runtime and gotten rid of the "get the NuGet package to get the ref" they've dropped it from theirs, but it's showing up as "this API doesn't come from this directory anymore" noise.

Reply:
is there any value for keeping it in the diff here ?

Reply:
I don't think so, no.